### PR TITLE
Edit sidebar

### PR DESCRIPTION
**Pull request by Amal-Mousa**

### CODE REVIEW

File: src/components/Sidebar/index.tsx [hunk 1]
```diff
@@ -81,43 +59,42 @@ const Sidebar = () => {
...
+              <NavLink to="/projects/addProject">
+                <IconButton onClick={handleOpen} sx={{ marginLeft: '2rem' }}>
+                  <Add sx={{ fontSize: 24, color: 'custom.white' }} />
+                </IconButton>
+                <AddProjectModal open={open} handleClose={handleClose} />
               </NavLink>
             </ProjectTextItem>
-            {projects.slice(0, 5).map((project) => (
+            {projects.slice(0, 3).map((project) => (
               <NavLink to={`/project/${project.project_id}`} style={{ textDecoration: 'none' }}>
```
Comment (nada-abuzaid) [inline src/components/Sidebar/index.tsx:98]: remove the NavLink and set the path into list item button directly!

File: src/components/Sidebar/index.tsx [hunk 2]
```diff
@@ -81,43 +59,42 @@ const Sidebar = () => {
...
           </List>
-          <Divider />
-          <List sx={{ fontSize: '1.1rem', paddingLeft: '0.4rem' }}>
-            <ProjectTextItem sx={{ color: 'custom.white' }}>
-              <NavLink to="/myproject">
+          <Divider sx={{ bgcolor: 'custom.white', opacity: '.2' }} />
+          <List
+            disablePadding
+            sx={{
+              p: 1, display: 'flex', flexDirection: 'column', gap: '0.8rem',
```
Comment (nada-abuzaid) [inline src/components/Sidebar/index.tsx:83]:
```suggestion
              p: 1, display: 'flex', flexDirection: 'column', gap: '0.6rem',
```

File: src/components/Sidebar/index.tsx [hunk 3]
```diff
@@ -81,43 +59,42 @@ const Sidebar = () => {
...
+            }}
+          >
+            <ProjectTextItem sx={{ display: 'flex', justifyContent: 'space-between', color: 'custom.white' }}>
+              <Typography sx={{ color: 'custom.white', fontSize: '18px' }}>
```
Comment (nada-abuzaid) [inline src/components/Sidebar/index.tsx:87]:
```suggestion
              <Typography sx={{ color: 'custom.fontGray', fontSize: '16px' }}>
```

File: src/components/Sidebar/index.tsx [hunk 4]
```diff
@@ -81,43 +59,42 @@ const Sidebar = () => {
...
-                <ListItem disablePadding>
-                  <ListItemButton>
-                    <ListItemIcon sx={{ minWidth: '42px' }}>
-                      {nav.icon}
-                    </ListItemIcon>
-                    <ListItemTextItem primary={nav.title} />
-                  </ListItemButton>
-                </ListItem>
-              </NavLink>
+          <Box sx={{ px: 2.5, py: 3, display: 'inline-flex' }}>
```
Comment (nada-abuzaid) [inline src/components/Sidebar/index.tsx:62]:
```suggestion
          <Box sx={{ px: 2.5, py: 2, display: 'inline-flex' }}>
```

File: src/components/Sidebar/index.tsx [hunk 5]
```diff
@@ -2,21 +2,16 @@ import { useState, useEffect } from 'react';
...
+  Typography,
 } from '@mui/material';
-import HomeIcon from '@mui/icons-material/Home';
-import FormatListNumberedIcon from '@mui/icons-material/FormatListNumbered';
-import AccountCircleIcon from '@mui/icons-material/AccountCircle';
 import { Add } from '@mui/icons-material';
+import { MdLogout } from 'react-icons/md';
 import { NavLink, useNavigate } from 'react-router-dom';
-import ReactLogo from '../../assets/logo.svg';
 // import PermanentAppBar from '../AppBar';
```
Comment (nada-abuzaid) [inline src/components/Sidebar/index.tsx:15]: remove it

File: src/components/Signup/index.tsx [hunk 6]
```diff
@@ -4,7 +4,7 @@ import {
   BodyContainer, CustomContainer, CustomTypography1,
   CustomTypography2, FormContent, DontHaveAcount, SigninText,
 } from './signup.styled';
-import ReactLogo from '../../assets/logo.svg';
+import ReactLogo from '../../../public/assets/logo.svg';
```
Comment (nada-abuzaid) [inline src/components/Signup/index.tsx:7]:
```suggestion
import ReactLogo from '/assets/logo.svg';
```

File: src/components/WelcomeCard/index.tsx [hunk 7]
```diff
@@ -3,7 +3,7 @@ import {
   CardActions,
   CardContent,
 } from '@mui/material';
-import myImage from '../../assets/welcome.svg';
+import myImage from '../../../public/assets/welcome.svg';
```
Comment (nada-abuzaid) [inline src/components/WelcomeCard/index.tsx:6]:
```suggestion
import myImage from '/assets/welcome.svg';
```

File: src/fake.ts [hunk 8]
```diff
@@ -132,4 +131,11 @@ const data = [
   },
 ];
 
-export default data;
+const account = {
+  displayName: 'Nada Abuzaid',
+  email: 'nada@gmail.com',
+  photoURL: '../public/assets/avatars/avatar_default.jpg',
```
Comment (nada-abuzaid) [inline src/fake.ts:137]:
```suggestion
  photoURL: '/assets/avatars/avatar_default.jpg',
```

File: src/components/Common/ListItem.tsx [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+        color: 'custom.white',
+        backgroundColor: 'custom.selected',
+        fontWeight: 'fontWeightBold',
+      },
+      '&:hover': { backgroundColor: 'custom.selected' },
+    }}
+  >
+    {icon ? <StyledNavItemIcon>{icon && icon}</StyledNavItemIcon> : (
+      <StyledNavItemIcon>
+        <MdDashboard style={{ fontSize: 22, color: 'custom.white' }} />
```
Comment (nada-abuzaid) [inline src/components/Common/ListItem.tsx:47]:
```suggestion
        <MdDashboard style={{ fontSize: 18, color: 'custom.white' }} />
```

File: src/components/Login/index.tsx
Comment (nada-abuzaid) [inline src/components/Login/index.tsx:1]: in the `src/components/Login/LoginForm/index.tsx`
line 121:
```JS
          <LoginImage src="/assets/todo.svg" alt="login" />
```

**Review by nada-abuzaid — CHANGES_REQUESTED** [review]

**Review by nada-abuzaid — APPROVED** [review]

**Review by Mohammed-Sbeata — APPROVED** [review]
BRAVOOOO NADA
👏👏👏👏

**Review by KhaledToha — APPROVED** [review]

**Pull request merged**